### DISCUSSION TITLE
Version diff pointless when document is too long and diff is only 1 or 2 lines.

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#2914] [Feature Requests & Ideas]
<img width="1326" alt="image" src="https://github.com/payloadcms/payload/assets/5801015/64e18f20-3abb-407e-8478-65eb15202fbd">

Consider collapse the same parts like github?

**Comment by DanRibbens** [comment]
> Consider collapse the same parts like github?

This is a great idea. I would love to see a community PR for this.
